code/game/gamemodes/gang/gang.dm

```diff
@@ -1,7 +1,7 @@
 //gang.dm
 //Gang War Game Mode
 
-var/list/gang_name_pool = list("Clandestine", "Prima", "Zero-G", "Max", "Blasto", "Waffle", "North", "Omni", "Newton", "Cyber", "Donk", "Gene", "Gib", "Tunnel", "Diablo", "Psyke", "Osiron", "Sirius", "Sleeping Carp")
```

Comment:
Might as well remove the gang, too, otherwise people will wonder why the Sleeping Carp gang doesn't give you the Sleeping Carp scroll